relayer/ics24.go

```diff
@@ -52,11 +52,16 @@ func (c *Chain) AddPath(clientID, connectionID string) error {
 
 // ValidateFull returns errors about invalid identifiers as well as unset path variables for the appropriate type
```

Comment:
Maybe update the godoc here to clarify behavior for zero client/connection ID.

Reply:
updated the godoc for all three methods that are being utilized in adding a `Path` to a `Chain`